packages/react-devtools-core/src/standalone.js

```diff
@@ -151,7 +153,16 @@ function startServer(port = 8097) {
     var backendFile = fs.readFileSync(
       path.join(__dirname, '../build/backend.js')
     );
-    res.end(backendFile + '\n;ReactDevToolsBackend.connectToDevTools();');
+    var devToolsConfig = {
+      host: process.env.DEVTOOLS_HOST,
+      port: process.env.DEVTOOLS_WEBSOCKET_PORT,
+      websocketProtocol: process.env.DEVTOOLS_WEBSOCKET_PROTOCOL,
+    };
+    var backendFileWithInit = backendFile +
+      '\n;ReactDevToolsBackend.connectToDevTools(' +
+      JSON.stringify(devToolsConfig) +
```

Comment:
@david-davidson This approach seems like it would play nicely with the request-header-reading code I proposed in https://github.com/facebook/react-devtools/pull/1220

What do you think of having fallbacks for `host` and `port` based on the request `Host` header when no `process.env.DEVTOOLS_HOST` or `process.env.DEVTOOLS_WEBSOCKET_PORT` are supplied?